src/taxman.py

```diff
@@ -137,7 +137,14 @@ def _evaluate_taxation_GERMANY(
                         f"({sc.op.__class__.__name__})"
                         for sc in sold_coins
                     )
-                    tx = transaction.TaxEvent(taxation_type, taxed_gain, op, remark)
+                    tx = transaction.TaxEvent(
+                        taxation_type,
+                        taxed_gain,
+                        op,
+                        total_win,
```

Comment:
You swapped sell and buy price. `total_win` is the sell_price and should come second.

We might want to rename `total_win` to avoid further confusion.

Reply:
The swapping should be irrelevant now that there's only the "sell price" in the CSV. Also, I renamed `total_win` to `sell_price` or would you suggest another naming? :)